app/models/user.rb

```diff
@@ -6,8 +6,8 @@ class User < ApplicationRecord
   validates :email, presence: true, length: {maximum: Settings.email_max},
     format: {with: VALID_EMAIL_REGEX}, uniqueness: {case_sensitive: false}
   has_secure_password
-  validates :password, length: {minimum: Settings.pass_min}
-
+  validates :password, presence: true, length: {minimum: Settings.pass_min},
+    allow_nil: true
```

Comment:
why do you use  allow_nil: true ?

Reply:
allow_nil cho phép bỏ qua validation khi giá trị là nil, ở đây dùng để update thông tin user mà có thể để trống password.